packages/development-proxy/mixin.core.js

```diff
@@ -26,11 +24,12 @@ class ProxyMixin extends Mixin {
     };
 
     if (typeof proxyConfig === 'string') {
-      if (!checkUrl(proxyConfig)) {
-        console.warn(
-          `The proxy target '${proxyConfig}' is invalid, disabling feature`
-        );
-
+      if (proxyConfig === '') {
```

Comment:
If a project has no proxy config at all, which value would we get here then?

Reply:
I don't think that we will get that far, because we have an [early exit](https://github.com/xing/hops/pull/838/files/f9d63985c79fdd7408ed7e1dcb5a6202940793e1#diff-24db1b4dc461fdb7ce25f11f261d6020R14) if the property `proxyConfig` is falsy.